### PR TITLE
Issue 135 - display xp payout on fights

### PR DESCRIPTION
**Pull request by Nizzypal**
### All Submissions

* [ ] Can you post a screenshot of your changes (if applicable)?

### New Feature Submissions

* [ ] Does this relate to an existing issue, or has it been talked about prior to being done (if a major change)?

### Notes

Please note, your code must pass all tests and lint checks before it can be merged.

### PR Description

(insert description here)

### CODE REVIEW

File: frontend/src/views/Combat.vue [hunk 1]
```diff
@@ -137,6 +140,7 @@ export default {
       intervalMinutes: null,
       timeSeconds: null,
       timeMinutes: null,
+      fightXpGain: 32,
```
Comment (seiyria) [inline frontend/src/views/Combat.vue:143]: XP is not guaranteed to be 32; this needs to be correctly calculated - if I remember right it's from 14-17, but that could be clarified.

File: frontend/src/views/Combat.vue [hunk 2]
```diff
@@ -286,10 +290,24 @@ export default {
         this.error = e.message;
       }
     },
+
     formattedSkill(skill) {
       const skillBalance = Web3.utils.fromWei(skill, 'ether');
       return `${new BN(skillBalance).toFixed(6)} SKILL`;
     },
+
+    onChooseWeapon(targetToFight) {
```
Comment (seiyria) [inline frontend/src/views/Combat.vue:299]: this function is confusingly named

File: frontend/src/views/Combat.vue (hunk 1 above)
Comment (Nizzypal) [reply frontend/src/views/Combat.vue:143]: who should we ask... i saw that number in cryptoblades.sol tho it looks like its some sort  of  special case
```
    function migrateTo_1ee400a() public {
        require(hasRole(DEFAULT_ADMIN_ROLE, msg.sender), "Not admin");

        fightXpGain = 32;
    }
```

File: frontend/src/views/Combat.vue (hunk 2 above)
Comment (Nizzypal) [reply frontend/src/views/Combat.vue:299]: right, yeah I did thename when the function was still an  event handler for an emit... it should be  changed.

**Comment by seiyria** [comment]
So if I were doing this, I would create a bold yellow text, position it absolutely over the art but near the bottom (above the button), instead of adding another prop to buttons.

We'll also need to get the exact amount of XP needed for this, but that you'll probably need some help getting as I'm not entirely sure how it's done yet.

**Comment by Nizzypal** [comment]
> So if I were doing this, I would create a bold yellow text, position it absolutely over the art but near the bottom (above the button), instead of adding another prop to buttons.
> 
> We'll also need to get the exact amount of XP needed for this, but that you'll probably need some help getting as I'm not entirely sure how it's done yet.

who do you think can we ask reg the exp?

**Review by seiyria — APPROVED** [review]

**Pull request merged**